include/ringmesh/basic/common.h

```diff
@@ -43,6 +43,9 @@
 #include <ringmesh/ringmesh_config.h>
 #include <ringmesh/ringmesh_export.h>
 
+#include <ringmesh/basic/range.h>
+#include <ringmesh/basic/types.h>
```

Comment:
Other includes are done line 117. Put yours at the same location.

Reply:
Why not put all includes in the top of the file as for all other files?